routes/user/userSecure.js

```diff
@@ -196,13 +197,19 @@ router.put('/', async (req, res) => {
                 if (updUser) {
                     await UserInfo.updateOne({_id: updUser._id}, req.body)
                     .then(async function (data, err){
+                        const teamToAdd = async () => {
+                            await TeamInfo.updateOne({_id: req.body.teamID}, {$push: {players: updUser._id}});
+                        }
+                        teamToAdd();

```

Comment:
does this need await?

Reply:
literally no clue but it worked? I thought it did but maybe not?

Reply:
"no it does" - aidan